dandiapi/settings.py

```diff
@@ -61,6 +61,8 @@ def before_binding(configuration: Type[ComposedConfiguration]):
     DANDI_DOI_API_PREFIX = values.Value(environ=True)
     DANDI_DOI_PUBLISH = values.BooleanValue(environ=True, default=False)
 
+    DANDI_VALIDATION_JOB_INTERVAL = values.IntegerValue(environ=True, default=60)
```

Comment:
Is that default 60 seconds or 60 minutes?

Reply:
Seconds. I added this so that the dandi-cli tests can use it to set the validation interval to something very short so they don't need to idle for a minute waiting for validation.